src/js/actions/index.js

```diff
@@ -18,7 +18,7 @@ import {
   FETCH_REASONCODES,
   FETCH_REQUISITION_STATUS_CODES,
   FETCH_SESSION_INFO,
-  FETCH_SHIPMENT_STATUS_CODES,
+  FETCH_SHIPMENT_STATUS_CODES, FETCH_SHIPMENT_TYPES,
```

Comment:
This should be on the next line for readability.

Reply:
yeah, this was done automatically - this is one of the "not deleted console.logs" I mentioned yesterday - I was just pushing the code late at night and didn't have the energy to fix those then, but wanted to push the code so that it gets your initial review guys, before Artur is out today.